openwillis/measures/video/preprocess_video.py

```diff
@@ -713,22 +706,21 @@ def prep_face_clusters_for_output(
             facedata_df.cluster==cluster_idx
         ]
         
-        # clean face data
-        cluster_df = filter_cluster_segments(
+        face_bbox_df = create_single_face_output(
             cluster_df,
             min_frames_face_present,
             frames_per_row,
-            fps
+            fps,
+            bbox_cols,
+            interpolate=interpolate
         )
 
```

Comment:
```suggestion
        if face_bbox_df.empty:
            continue
```

To handle case where `face_bbox_df` is empty for a cluster